plugins/inputs/bind/bind_test.go

```diff
@@ -0,0 +1,200 @@
+package bind
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/influxdata/telegraf/testutil"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBindJsonStats(t *testing.T) {
+	ts := httptest.NewServer(http.FileServer(http.Dir("testdata")))
+	defer ts.Close()
+
+	b := Bind{
+		Urls: []string{ts.URL + "/json/v1"},
```

Comment:
Can do, but this will require adding some potentially large test files to the repo, which I presumed you wanted to try to avoid.